client/my-sites/pages/page/index.js

```diff
@@ -73,6 +73,8 @@ class Page extends Component {
 		recordMoreOptions: PropTypes.func.isRequired,
 		recordPageTitle: PropTypes.func.isRequired,
 		recordEditPage: PropTypes.func.isRequired,
+		// Adding new make homepage item
```

Comment:
This comment is not needed.  We will look back at the version control history when we need to know what this is for and where it came from.

Reply:
Sounds good. Should I go into my branch and remove all comments?